sql/driver/conn.go

```diff
@@ -20,8 +20,6 @@ package driver
 import (
 	"database/sql/driver"
 	"fmt"
-
```

Comment:
this isn't new to this PR, just noticing it now. We're importing database/sql/driver from within the driver package. This works, but it feels dangerous and confusing.